lib/springboard/client/body.rb

```diff
@@ -7,6 +7,7 @@ class Client
     #
     # @see http://rdoc.info/github/intridea/hashie/Hashie/Mash See the Hashie::Mash docs for usage details
     class Body < ::Hashie::Mash
+      self.disable_warnings
```

Comment:
The `self` here is unnecessary.

Also, rather than require a newer `hashie` version I think it'd bet better to just conditionally disable the warnings if the hashie gem supports it. Something like:

```ruby
disable_warnings if respond_to?(:disable_warnings)
```